openstack_dashboard/settings.py

```diff
@@ -153,15 +153,6 @@
     },
 ]
 
-# Workaround for template loading of titanium theme
-if not os.path.exists('/opt/branding/applied'):
-    TEMPLATES[0]['DIRS'] = ['/usr/share/openstack-dashboard/'
-                            'openstack_dashboard/themes/titanium/templates',
-                            os.path.join(ROOT_PATH, 'templates'), ]
-elif os.path.exists('/opt/branding/applied/templates'):
-    TEMPLATES[0]['DIRS'] = ['/opt/branding/applied/templates',
-                            os.path.join(ROOT_PATH, 'templates'), ]
```

Comment:
I believe we still need the elif part of the above code, it is used for our custom branding feature to get around horizon disallowing template dirs outside of horizon

Reply:
Will fix and upload a new version